src/views/app.styles.js

```diff
@@ -46,14 +46,14 @@ const useAppStyles = createUseStyles((theme) => ({
   "@global": {
     "*": {
       boxSizing: "border-box",
```

Comment:
Maybe is also a good idea to move the `border-box`?

Reply:
box-sizing is not inherited